src/config/navigationConfig.js

```diff
@@ -37,6 +39,14 @@ export const linksHeader = atom({
       filter: 'Womens',
       url: '/womens',
     },
+    {
+      name: 'Womens and Accessories Even and Odd',
+      type: 'filterComplex',
+      filter: '',
+      moreComplexFilter:
```

Comment:
should we adjust the attribute name to be something clearer like `rawFilter` and make that the type as well?